portfolio.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css" />
+    <link rel="stylesheet" href="./font-awesome-4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <div id="image-container">
+    <img id="ellipses" src="./images/Top-circles.png" alt="circles">
+  </div>
+  <nav>
+    <p class="placeholder">John Doe</p>
+    <button id="menu">
+      <span class="line"></span> 
+      <span class="line"></span>
+      <span class="line"></span>
+    </button>
+  </nav>
+  <header>
+    <h1>Justo Rago</h1>
+    <h2>Software Developer</h2>
+  </header>
```

Comment:
- [ ] Would you please include this part `name and the title`  in the headline section as it is part of the headline section for semantic and accessibility reasons.